NSS-Spontaneous-Spoonbills/Sprint3
# Faker SQL Scripts

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes #23 

## Description of Proposed Changes
New directory: sql
New file: bangazon.sql

## Steps to Test
1. checkout branch
1. run virtual machine
1. see that sql directory has a file named bangazon.sql
1. make all necessary migrations 
1. Open DB Browser with db.sqlite3
1. Navigate to 'Execute SQL'
1. Run Script
1. Browse website_customer, website_product_type, & website_payment_type make sure it inserted successful
1. Use :shipit: if completed
1. AND ONLY  :shipit: 


```
git fetch --all
git checkout js_sql_scripts
```

## Impacted Areas in Application

**New Files**:
sql/bangazon.sql

**Modified Files**:


**Deleted Files**:



## Mentions @username

Tag users that need to review this code

## Testing


## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
